src/main/scala/org/scalajs/tools/tsimporter/sc/Definitions.scala

```diff
@@ -7,7 +7,7 @@ package org.scalajs.tools.tsimporter.sc
 
 import scala.language.implicitConversions
 
-import scala.collection.mutable._
+import scala.collection.mutable.{ Set => _, _ }
```

Comment:
Since you're modifying this line, you might as well really fix it and removing the wildcard import. Instead, explicitly import what is actually used. Wildcard imports in the `mutable` package are evil, we shouldn't do that.

Reply:
Done in a separate commit